### ISSUE TITLE
Unable to scroll through data stream in Data: text area wrapper

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/center-and-variability#104]
**Test device**
MacBook Air (m1 chip)

**Operating System**
12.3

**Browser**
Safari//Chrome

**Problem description**
For https://github.com/phetsims/qa/issues/789
I am unable to scroll through the data stream after making changes in the sim. I can only see information about the model.timeProperty.
To make sure this wasn't a general issue, I tested with Density in master and had no issues. There, the data stream pauses once I click on it and allows for scrolling. 

**Visuals**

https://user-images.githubusercontent.com/87318828/159739502-0d2a5b3d-aaf8-49ea-b771-4335ca58b7d4.mp4

**Comment by KatieWoe** [comment]
I think this is because new out put is constantly being created, which scrolls back down to the bottom. May not exactly be a bug, but definitely non-ideal.

**Comment by samreid** [comment]
This patch only autoscrolls to the bottom if you were already at the bottom (like how the data streaming works in the chrome dev tools console). It seems OK in my limited tests.

<details>

```diff

Subject: [PATCH] Add SimVersion, see https://github.com/phetsims/center-and-variability/issues/104
---
Index: data-stream/index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/data-stream/index.html b/data-stream/index.html
--- a/data-stream/index.html	(revision 52e04749c3477daec340a79949e0ca884da02aa5)
+++ b/data-stream/index.html	(date 1682034123982)
@@ -110,10 +110,23 @@
   // Update the DOM intermittently so performance isn't too disturbed
   var lastEvent = '';
   setInterval( function() {
-    var event = eventStack.map( function( event ) { return JSON.stringify( event, null, 2 ); } ).join( '\n' );
+    var event = eventStack.map( function( event ) {
+      return JSON.stringify( event, null, 2 );
+    } ).join( '\n' );
+
     if ( lastEvent !== event ) {
+
+      // Check if the textarea is already scrolled to the bottom before updating
+      var isAtBottom = textarea.scrollTop + textarea.clientHeight >= textarea.scrollHeight - 10;
+
       textarea.textContent = event;
-      textarea.scrollTop = textarea.scrollHeight;
+
+      // Only autoscroll if the textarea was already at the bottom
+      console.log( isAtBottom );
+      if ( isAtBottom ) {
+        textarea.scrollTop = textarea.scrollHeight;
+      }
+
       lastEvent = event;
     }
   }, 120 );

```

</details>

I bet @zepumph would be the best judge of whether this is an equitable solution or whether it would have awkward corner cases, but we can also do more testing on my subteam.

**Comment by marlitas** [comment]
@samreid I tried this out and it won't let me scroll up in the data stream... Seems strange?

But I don't know the use cases for this wrapper, because I personally would love to be able to pause it and actually read the data, but that seems like functionality that isn't important here.

**Comment by zepumph** [comment]
Not sure what the best solution is here. I think we would want more design thought rather than treating it as a bug. The issue here is that the text area is updating every single time there is another data event from the sim. In the past we used highFrequencyEvents:false to make it more usable. Perhaps we should instead get rid of this wrapper (it kinda sucks anyways). Otherwise. . . 
* we could have it update less often.
* We could have a "take snapshot" button that shows what the data even was when you pressed it in another pane.
* It seems like high-frequency events are the cause of the usability trouble, do we want to investigate bringing that feature back? (I don't think so)
* We could realize how much better the console log is, and (again), just delete it.

**Comment by zepumph** [comment]
In my opinion. This issue should be moved to phet-io-wrappers and deferred until we actually create a designed, "wrapper index" landing page for PhET-iO.

**Comment by marlitas** [comment]
I agree with @zepumph . I don't think this fix feels helpful and it seems like there are larger questions at play here that are bigger than CaV.

**Comment by samreid** [comment]
Everything above sounds good, but I would also probably +1 for deleting it on integrating into studio.  I'll move it to phet-io-wrappers now.
